optaplanner-docs/src/main/asciidoc/QuickStart/CloudBalancingDomainModel/CloudBalancingDomainModel-section.adoc

```diff
@@ -120,14 +125,14 @@ The `CloudBalance` class has a [path]_@PlanningSolution_
  annotation.
 
 * It holds a list of all computers and processes.
-* It represents both the planning problem and (if it's initialized) the planning solution.
+* It represents both the planning problem and (if it is initialized) the planning solution.
 
-. Planner needs to retrieve the collection of processes that it can change, therefore we annotate the getter `getProcessList()` with ``@PlanningEntityCollectionProperty``.
+. The class holds a collection of processes that Planner can change. We annotate the getter `getProcessList()` with ``@PlanningEntityCollectionProperty``, so that Planner can retrieve the processes that it can change. To save a solution, Planner initializes a new instance of the class with the list of changed processes.
 
 . The `CloudBalance` class also has a `@PlanningScore` annotated property ``score``, which is the `Score` of that solution in its current state.
-Planner automatically updates it when it calculates a `Score` for a solution instance and therefore it needs a setter.
+Planner automatically updates it when it calculates a `Score` for a solution instance; therefore, this property needs a setter.
 
-. Especially for score calculation with Drools, the property `computerList` needs to be annotated with a `@ProblemFactCollectionProperty` so the computers are known to it.
+. Especially for score calculation with Drools, the property `computerList` needs to be annotated with a `@ProblemFactCollectionProperty` so that Planner can retrieve a list of computers (problem facts) and make it available to the Drools engine.
```

Comment:
retrieve a list of -> retrieve the list of

Reply:
changed, thanks